src/pg-resource.ts

```diff
@@ -83,7 +83,7 @@ export function mkCreate<T extends ModelBase>({
   db,
   tableName,
 }: ResourceOptions) {
-  return async function create<C extends TypedBody<Omit<T, 'id'>>>(
+  return async function create<C extends TypedBody<Except<T, 'id'>>>(
```

Comment:
Let's just use typescript's built-in `Omit`.

Reply:
Added in b55c495